.github/workflows/postsubmit.yml

```diff
@@ -39,7 +39,7 @@ jobs:
         with:
           cluster_name: build-kubeflow
       - name: Run tests
-        run: |
+        run: >
```

Comment:
have you tested this out? I believe backslashes would be required even with the folder style.

Reply:
Will update this but this still must work

Reply:
https://stackoverflow.com/questions/59954185/github-action-split-long-command-into-multiple-lines